tasks/parse-pdr/schemas/output.json

```diff
@@ -31,7 +31,9 @@
                 "name": { "type": "string" },
                 "fileSize": { "type": "number" },
                 "checksumType": { "type": "string" },
-                "checksumValue": { "type": "number" }
+                "checksumValue": {
```

Comment:
According to the JSON Schema docs, it seems like `"type": ["string", "number"]` would be correct here.

http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.1.1